Antidote/QRScannerAimView.h

```diff
@@ -0,0 +1,12 @@
+//
+//  QRScannerAimView.h
+//  Antidote
+//
+//  Created by Nikolay Palamar on 12/04/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface QRScannerAimView : UIView
+@end
```

Comment:
Aim view doesn't work for me. I've checked on iOS 8.2 Simulator and device iOS 7.1.2. I see no dashes or something similar.